libnmstate/ifaces/ifaces.py

```diff
@@ -194,6 +197,53 @@ def _validate_vlan_mtu(self):
                         f"MTU({base_iface.mtu})"
                     )
 
+    def _validate_sriov_mtu(self):
```

Comment:
Please split this method so it will be clear what is the high level logic without entering all the details.
E.g. building the pf-vf tree is one thing, calculating the max mtu is another and setting it on the PF a 3rd.

Reply:
For detail,

You may change `EthernetIface.__init__()` to detect whether certain NIC is VF and set the `EthernetIface.parent` property to its PF.

Then rename `Ifaces._validate_vlan_mtu()` to `Ifaces._validate_child_mtu` with small modification, go get consistant
behavior like we did for VLAN/VxLAN.